de.dlr.sc.virsat.model.extension.cefx.test/src/de/dlr/sc/virsat/model/extension/cefx/migrator/Migrator1v2Test.java

```diff
@@ -0,0 +1,50 @@
+/*******************************************************************************
+ * Copyright (c) 2008-2019 German Aerospace Center (DLR), Simulation and Software Technology, Germany.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+package de.dlr.sc.virsat.model.extension.cefx.migrator;
+
+// *****************************************************************
+// * Import Statements
+// *****************************************************************
+
+
+import de.dlr.sc.virsat.model.dvlm.DVLMFactory;
+import de.dlr.sc.virsat.model.dvlm.Repository;
+
+// *****************************************************************
+// * Class Declaration
+// *****************************************************************
+
+import org.junit.Test;
+
+/**
+ * Auto Generated Class inheriting from Generator Gap Class
+ * 
+ * This class is generated once, do your changes here
+ * 
+ * VirSat DLR CEF Concept for extended Product Structures
+ * 
+ */
+public class Migrator1v2Test extends AMigrator1v2Test {		
+	
+	@Test
+	public void testMigrator1v2() {
+		Migrator1v2 testMigrator1v2 = new Migrator1v2();
+		
+		Repository repository = DVLMFactory.eINSTANCE.createRepository();
+		repository.getActiveConcepts().add(conceptMigrateFromRepository);
+		
+		//TODO: Setup test instances
```

Comment:
Please remove these TODO statements... Otherwise checkstyle will fail

Reply:
I removed it.